src/pages/en/guides/markdown-content.md

```diff
@@ -146,8 +44,10 @@ const { content } = Astro.props;
   </body>
 </html>
 ```
+### The content prop
```

Comment:
```suggestion
### Props
```

Reply:
Hmm... normally I like shorter! But, I do kind of like reinforcing that it's "the content prop".

Other opinions??

Reply:
I also feel “Props” would be a sightly different topic. This section is specifically about `content` right?

Reply:
Yeah, so let's go with The Content Prop for now.

(This is all about getting it ready and making the best choices we can until Fred reads it anyway, to make his life easier! He'll have final say!)